desktop/doctor/src/index.ts

```diff
@@ -38,24 +38,28 @@ export type Healthcheck = {
   key: string;
```

Comment:
The changes here seem unrelated. Could you split this off into a separate PR if you want to submit them? :)